src/deadline/client/ui/widgets/host_requirements_tab.py

```diff
@@ -693,6 +802,67 @@ def _set_remove_button_for_first_item(self):
             first_item = self.value_list_widget.itemWidget(self.value_list_widget.item(0))
             first_item.remove_button.setEnabled(True)
 
+    def name(self) -> str:
```

Comment:
Similar note here about `@property` and `@setter` methods

Reply:
Good idea, working on that now